app/views/how-tos/making-pages.html

```diff
@@ -17,29 +17,42 @@ <h1>
         {{ pageName }}
       </h1>
 
-      <p class="nhsuk-lede-text">Use your HTML text editor to make pages.</p>
+      <p class="nhsuk-lede-text">Use your code editor to make pages.</p>
 
-      <p>Create all your HTML pages in the <strong>/app/views</strong> folder.</p>
+      <p>Create all your pages in the <code>/app/views</code> folder.</p>
 
-      <p>All pages need the <strong>.html</strong> extension.</p>
+      <p>The file names must end in <code>.html</code> – this is called the file extension.</p>
 
-      <p>Pages within <strong>/app/views</strong> will be automatically available in your web browser.</p>
+      <p> It’s a good idea to name the files using a lowercase version of the page title, with spaces replaced by hyphens. For example, <code>find-a-gap.html</code>.</p>
```

Comment:
```suggestion
      <p>It’s a good idea to name the files using a shortened, lowercase version of the page title, with spaces replaced by hyphens. For example, <code>find-a-gp.html</code>.</p>
```

I'd usually remove stop words from the titles, and reasonably shorten them.